src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -210,10 +210,16 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.ErrorIndexStartupBehavior", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public IndexStartupBehavior ErrorIndexStartupBehavior { get; set; }
 
+        [Description("Location of NuGet packages cache")]
+        [DefaultValue("Packages/NuGet")]
```

Comment:
What is the base directory? shouldn't this be under indexes?

Reply:
Index compiler is static, loaded assemblies are also application-wide. So it is in the main directory. I think it is correct?

Reply:
yes